api/graphql/mutations/project.js

```diff
@@ -23,7 +23,11 @@ async function getModeratedProject (userId, projectId) {
   }
 
   if (project.relations.user.id !== userId) {
-    throw new GraphQLYogaError("You don't have permission to moderate this project")
+    const responsibilities = await Responsibility.fetchForUserAndGroupAsStrings(userId, project.id)
```

Comment:
broken, but we can ignore this, projects are posts right now and permissions are handled differently for those